src/sentry/models/group.py

```diff
@@ -43,6 +43,7 @@
     PriorityChangeReason,
     get_priority_for_ongoing_group,
 )
+from sentry.models.commit import Commit
```

Comment:
I think this should be imported in the `get_suspect_commit` method only probably